CONTRIBUTING.md

```diff
@@ -15,35 +24,42 @@ Did you find a bug in GeoBlacklight or interested in a new feature? Make sure to
     - Explain the actual behavior
     - Provide steps to reproduce the actual behavior
 
-### GeoBlacklight Software Versioning
-GeoBlacklight follows the practice of [Semantic Versioning](https://semver.org/) for software releases. The declared semantically versioned public API includes:
- - the GeoBlacklight-Schema
- - the [public GeoBlacklight Ruby codebase classes](http://www.rubydoc.info/github/geoblacklight/geoblacklight/master/frames)
- - the GeoBlacklight JavaScript interface
- - the GeoBlacklight view interface
+### Issue Assignment
+Issues are used in the GeoBlacklight community as a convenient way to document bugs, propose new functionality and enhancements, and discuss important code changes.
+
+- If someone is already assigned to an issue, that does not mean you can't contribute.
+
 
 ### Contributing code or documentation
 GeoBlacklight welcomes code and documentation contributions. You do *not* need to be a "GeoBlacklight Committer" to contribute code or documentation. We follow the [pull request](https://help.github.com/articles/using-pull-requests/) model for contributing on GitHub. GeoBlacklight uses a suite of tests to express its features and protect from bugs:bug:.
 
 When proposing major new features or changes that may introduce an API or schema change, please make sure to communicate with the [community](http://geoblacklight.org/connect) so the full implications are understood. Likely there are ways to introduce these changes in a backwards compatible way that others may be able to help with.
 
 #### Pull request overview
+1. If there isn't already an associated issue in the repository, create one.
 1. Fork it ( http://github.com/my-github-username/geoblacklight/fork )
```

Comment:
This is a suggested revision based on the language we used in the website/documentation guide. What I like about it is it's simpler (no git commands, which comfortable git users probably already know), and it clarifies the pull request process:

```
**Contributors:**

1. If there isn't already an associated issue in the repository, create one
1. Clone or fork the geoblacklight repository
1. Create a new feature branch and publish it
1. Make changes to the files
1. Commit your changes
1. Push to the new branch
1. Open a Pull Request to the **main** branch
1. Add GeoBlacklight-Developers as a requested reviewer
1. Mention the pull request in the associated issue

**GeoBlacklight-Developers:**

1. Review the Pull Request and merge changes to the **main** branch
```

Reply:
Much cleaner!

Reply:
@rmseifried Should this also replace the _Merging Changes_ section? Are we good with tossing out that other information? I have an idea I will include in upcoming commit...

Reply:
> 1. Open a Pull Request to the **main** branch

Do we want to get into the different branches here? What if someone is contributing a change to 4.x?

Reply:
> Do we want to get into the different branches here? What if someone is contributing a change to 4.x?

That's a good point! Perhaps instead of "**main** branch" it could be "appropriate branch, like **main**, **release-4.x**, etc."